src/gui/sysPartNumber.cpp

```diff
@@ -77,9 +77,7 @@ const char* FurnaceGUI::getSystemPartNumber(DivSystem sys, DivConfig& flags) {
         }
       break;
     }
-    case DIV_SYSTEM_AMIGA:
-      return "Amiga";
-      break;
+    //Amiga -> Paula?
```

Comment:
No. "Amiga" is correct.

Reply:
Further explanation:

In Amiga, two sound chips are involved in the generation of sound: Agnus, and Paula.

Paula outputs sound to the speakers and sends DMA requests to Agnus.
Agnus reads from memory and sends the output to Paula.